src/App/Types.hs

```diff
@@ -100,3 +101,5 @@ instance Semigroup OverrideDynamicAnalysisBinary where
 
 instance Monoid OverrideDynamicAnalysisBinary where
   mempty = OverrideDynamicAnalysisBinary mempty
+
+newtype FullFileUploads = FullFileUploads {unFullFileUploads :: Bool} deriving (Eq, Ord, Show, Generic)
```

Comment:
[optional] I think a more idiomatic way to do this would be something like `data FullFileUploads = FullFileUpload | LicenseSnippetUpload deriving (Eq, Ord, Show)`.

However, the other config values in the `Organization` type are `Bool`s and it might require a little more work for marshalling/unmarshalling so I think it's ok if you decide not to do this. I just wanted to call it out.